plugins/change.org/change_org

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+'''
+=head1 NAME
+
+Munin Plugin to grab signature count for change.org petition given its ID
+You need a valid API key for this petition
+
+=head2 CONFIGURATION
+
+# Sample:
+[change_org]
+	user nobody
```

Comment:
User and group selection depends on the distribution. Since "nobody/nogroup" should be the default setting, I would recommend to just remove these lines from the config example.